Edge10.CouchDb.Client/ViewParameters.cs

```diff
@@ -131,8 +131,8 @@ public override string CreateQueryString()
 				parameters.Add("inclusive_end", InclusiveEnd.Value.ToString().ToLower());
 			if (Skip.HasValue)
 				parameters.Add("skip", Skip.Value.ToString());
-			if (Key != null && Key.Any())
-				parameters.Add("key", PrepareArrayValue(Key));
+			if (Key != null)
+				parameters.Add("key", Key.FormattedValue);
```

Comment:
Do we need to account for a blank value still, ie if the new class is called with an empty array?